languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -46521,6 +46521,31 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Grammatik -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammatik">
+        <rule id="SOWOHL_UND" name="sowohl...und (als/wie auch)" default="temp_off">
+            <antipattern>
+                <token skip="-1">sowohl</token>
+                <token>als</token>
+                <token>auch</token>
+            </antipattern>
+            <antipattern>
+                <token skip="-1">sowohl</token>
+                <token>wie</token>
+                <token>auch</token>
+            </antipattern>
+            <pattern>
+                <token skip="-1">sowohl</token>
+                <marker>
+                    <token>und</token>
+                </marker>
+            </pattern>
+            <message>Die schriftsprachlich elegantere Form lautet 'sowohl ... als/wie auch'.</message>
```

Comment:
Bist du dir mit "sowohl... **wie** auch" sicher? Hast du dafür Quellen? In meinen Ohren klingt das nicht richtig...

Reply:
@udomai 
Gute Quellen, außer mich ;-), habe ich leider nicht... nur das hier: https://de.wiktionary.org/wiki/sowohl_%E2%80%A6_wie_auch
Aber, weil du sagst, das höre sich in deinen Ohren falsch an, habe ich es mal rausgenommen.
Ansonsten wäre es ready to merge